components/sections/build-section/index.tsx

```diff
@@ -0,0 +1,161 @@
+        width={188}
+        height={91}
+        className="absolute right-4 top-16 md:top-17.5 md:h-[81px] md:w-[167px] lg:right-0 lg:top-[102px] lg:h-[117px] lg:w-[226px] xl:top-[92px] xl:h-[136px] xl:w-[264px] 2xl:right-2 2xl:top-[81px] 2xl:h-[159px] 2xl:w-[328px]"
+      />
+    ),
+  },
+  {
+    title: "Discord",
+    description: "Chat with fellow developers and validators.",
+    // these widths are temp, waiting for copy adaptation
+    descriptionClassName: "lg:w-[240px] xl:w-[300px] 2xl:w-auto",
+    iconUri: "/assets/build/card-icons/discord.svg",
+    link: "#",
+    illustration: (
+      <Image
+        src={"/assets/build/illustrations/discord.svg"}
+        alt="Discord illustration"
+        width={123}
+        height={142}
+        className="absolute right-[62px] top-6 md:right-14 md:top-7.5 md:h-[133px] md:w-[115px] lg:right-13 lg:top-10.5 lg:h-52 lg:w-[180px] xl:top-6 xl:h-[235px] xl:w-[209px] 2xl:right-12 2xl:top-7.5 2xl:h-[231px] 2xl:w-[217px]"
+      />
+    ),
+  },
+];
+
+export default function BuildSection() {
+  return (
+    <section className="mt-28 flex flex-col items-center justify-center gap-8 self-stretch rounded-4xl p-2 sm:mt-30 sm:p-4 md:mt-39 lg:mt-32 xl:mt-44 2xl:mt-[152px] 2xl:gap-14 2xl:p-6">
+      <div className="flex flex-col items-center gap-4">
+        <div className="flex max-w-max items-center justify-center rounded-full bg-[#37D56D40] px-3 py-1 backdrop-blur-xl">
+          <span className="text-sm leading-[22px] text-[#37D56D] lg:text-base lg:leading-[25px]">
+            Developers
+          </span>
+        </div>
+        <div className="flex flex-col items-center gap-4 text-center">
+          <h2 className="bg-build-headline font-h1 text-3.5xl font-medium leading-[35px] -tracking-[0.64px] md:text-4xl md:leading-[39px] md:-tracking-[0.72px] lg:text-5xl lg:leading-[52px] lg:-tracking-[0.96px] xl:text-[56px] xl:leading-[61px] xl:-tracking-[1.12px] 2xl:text-[64px] 2xl:leading-[70px] 2xl:-tracking-[1.28px]">
+            Build the next <br /> disruptive DeFi App.
+          </h2>
+          <p className="font-light leading-[22px] text-neutral-100 lg:text-xl">
+            Osmosis empowers developers to <br className="md:hidden" />
+            innovate.
+          </p>
+        </div>
+      </div>
+      {/**stack */}
+      <div className="flex flex-col items-center gap-2 self-stretch xl:gap-4">
+        {/**row */}
+        <div className="flex flex-col items-center gap-2 self-stretch md:grid md:grid-cols-2 xl:gap-4">
+          {cards.slice(0, 2).map((props) => (
+            <BuildCard key={props.title} {...props} />
+          ))}
+        </div>
+        {/**row */}
+        <div className="flex flex-col items-center gap-2 self-stretch md:grid md:grid-cols-2 xl:gap-4">
+          {cards.slice(2, 4).map((props) => (
+            <BuildCard key={props.title} {...props} />
+          ))}
+        </div>
+      </div>
+    </section>
+  );
+}
+
+function BuildCard({
```

Comment:
Seems like a lot of cards are similar, build for example "Build", "Earn" and "Help" are pretty similar, maybe we can find a way to aggregate them in a single component

Reply:
ref in #30